pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/block/DataBlockValSet.java

```diff
@@ -36,29 +36,28 @@
  * TODO: Support MV
  */
 public class DataBlockValSet implements BlockValSet {
-  protected final FieldSpec.DataType _dataType;
-  protected final DataBlock _dataBlock;
-  protected final int _index;
-  protected final RoaringBitmap _nullBitMap;
+  private final DataType _dataType;
```

Comment:
i knew that DataBlock is a v2 concept. but other than that any specific reason we put these in the runtime module and the RowBasedBlockValSet in the core module? (i see DataBlock is still in core, imo might be eaiser to put everything together)

Reply:
I would say there is no specific reason. Ideally we should move all `BlockValSet` together, but currently they are spread over 3 places. Can be done in a separate PR